addons/point_of_sale/controllers/main.py

```diff
@@ -29,3 +32,55 @@ def print_sale_details(self, date_start=False, date_stop=False, **kw):
         pdf = request.env['report'].with_context(date_start=date_start, date_stop=date_stop).get_pdf(r, 'point_of_sale.report_saledetails')
         pdfhttpheaders = [('Content-Type', 'application/pdf'), ('Content-Length', len(pdf))]
         return request.make_response(pdf, headers=pdfhttpheaders)
+
+    @http.route('/pos/debugging_mail', type='json', auth="user")
+    def send_email(self, debug_type, date_string=None, data=None, **kwargs):
+        Mail_mail = request.env['mail.mail']
+        Ir_attachment = request.env['ir.attachment']
+        user_from = request.env['res.users'].search([("id", "=", request.session.uid)])
+        email_to = user_from.email
+
+        if not date_string:
+            date_string = json.loads(data).keys()[0] + "_" + datetime.now().strftime("%Y-%m-%d_%H_%M_%S")
+
+        subject = "[POS][DEBUG] Support " + date_string
+
+        result = None
+        mail_to_send = None
+        try:
+            mail_body = ("<p>Hello,</p>" +
+                         "<p>Please find in attachment the export of debug type [" + debug_type + "]</p>" +
+                         "<p>Best Regards</p>")
+
+            mail_to_send = Mail_mail.create({
+                'subject': subject,
+                'body_html': mail_body,
+                'email_to': email_to,
+                "email_from": user_from.email})
+
+            attachement = Ir_attachment.create({
+                'name': subject,
+                'datas_fname': date_string + '.txt',
+                'datas': str(data).encode('base64'),
+                'res_model': 'mail.mail',
+                'res_id': mail_to_send.id})
+
+            mail_to_send.write({'attachment_ids': [(6, 0, [attachement.id])]
+                                })
+
+            mail_to_send.send()
+            if not mail_to_send.mail_sent:
```

Comment:
`mail_sent` is not a field. The way to check an email was delivered is to use `_postprocess_sent_message` but I think it is out of the scope of this PR. If you could make an email without traceback but not deliver it, it means the server is not properly configured and no mail is sent.